crates/egui/src/containers/popup.rs

```diff
@@ -87,7 +87,8 @@ pub fn show_tooltip_at_pointer<R>(
 
         // Add a small exclusion zone around the pointer to avoid tooltips
         // covering what we're hovering over.
-        let mut exclusion_rect = Rect::from_center_size(pointer_pos, Vec2::splat(24.0));
+        let exclusion_size = Vec2::splat(32.0 * ctx.native_pixels_per_point().unwrap_or(1.0));
```

Comment:
Why would the pixel density of the display matter here?

Reply:
> Why would the pixel density of the display matter here?

On Windows, the mouse cursor size varies depending on `native_pixels_per_point`.
I don't know if the mouse cursor size varies on other OSes.